crates/core/src/surfnet/svm.rs

```diff
@@ -937,91 +944,177 @@ impl SurfnetSvm {
+                                    v0_msg
+                                        .address_table_lookups
+                                        .iter()
+                                        .map(UiAddressTableLookup::from)
+                                        .collect(),
+                                ),
+                            ),
+                        };
+
+                    let inner_instructions = OptionSerializer::Some(
+                        meta.inner_instructions
+                            .iter()
+                            .enumerate()
+                            .map(|(i, inner_ixs)| UiInnerInstructions {
+                                index: i as u8,
+                                instructions: inner_ixs
+                                    .iter()
+                                    .map(|ix| {
+                                        UiInstruction::Compiled(UiCompiledInstruction::from(
+                                            &ix.instruction,
+                                            Some(ix.stack_height as u32),
+                                        ))
+                                    })
+                                    .collect(),
+                            })
+                            .collect(),
+                    );
+
+                    let transaction = EncodedTransactionWithStatusMeta {
+                        transaction: EncodedTransaction::Json(UiTransaction {
+                            signatures: tx.signatures.iter().map(|s| s.to_string()).collect(),
+                            message: UiMessage::Raw(UiRawMessage {
+                                header,
+                                account_keys,
+                                recent_blockhash: tx.get_recent_blockhash().to_string(),
+                                instructions,
+                                address_table_lookups,
+                            }),
+                        }),
+                        meta: Some(UiTransactionStatusMeta {
+                            err: err.clone(),
+                            status: err.map(|e| Err(e)).unwrap_or(Ok(())),
+                            inner_instructions,
+                            log_messages: OptionSerializer::Some(meta.logs.clone()),
+                            return_data: OptionSerializer::None,
+                            compute_units_consumed: OptionSerializer::Some(
+                                meta.compute_units_consumed,
+                            ),
+                            pre_token_balances: OptionSerializer::None, // todo
+                            post_token_balances: OptionSerializer::None, // todo
+                            rewards: OptionSerializer::None,            // todo
+                            loaded_addresses: OptionSerializer::None,   // todo
+                            pre_balances: vec![],
+                            post_balances: vec![],
+                            fee: 0,
```

Comment:
Can you create an issue to track our need to start storing TransactionStatusMeta data with every block?